### DISCUSSION TITLE
Add Custom Cronjob on SO Master Node

### DISCUSSION THREAD

**Discussion by Xboarder56** [Security-Onion-Solutions/securityonion#4638] [Unsupported Versions] [answered]
What's the recommended process to add a custom cronjob for Security Onion 2.3.52? I noticed the crontab shows it's managed by salt. Can someone provide an example/steps to add a custom script to the crontab via salt?

**Comment by facyber** [comment]
Check `/opt/so/saltstack/default/salt/common/init.sls` file, there are defined regular or non-curator cronjobs as far as I see. If you want to make any modifications, do not modify that file, but make a copy of it in `/opt/so/saltstack/local/salt/common/` and then modify that file.

Try to search for few of those already defined to get familiar of where are the files and in what form are they, but as told, do not modify any of those files in `default` folders, but rather copy or create a new ones in `local` folder.

**Comment by TOoSmOotH** [comment, marked as answer]
Don't copy anything to local unless you really need to. 

If you look at the root crontab you will see:
# Lines below here are managed by Salt, do not edit

Just put your entries above this line and salt won't touch them.
